schemas/hs00_event_histogram.fbs

```diff
@@ -0,0 +1,37 @@
+
+file_identifier "hs00";
+
+table ArrayUInt   { value: [uint];    }
+table ArrayULong  { value: [ulong];   }
+table ArrayDouble { value: [ double]; }
+
+// Allows to use multiple data format for the arrays
+union Array {
+    ArrayUInt,
+    ArrayULong,
+    ArrayDouble
+}
+
+// Meta information for one dimension
+table DimensionMetaData {
+    size: uint;        // Size of the full histogram for this dimension
+    unit: string;      // Unit for this dimension
+    label: string;     // Label for this dimension
+    bins: Array;       // Binning information for this dimension
```

Comment:
I think this should be bin boundaries and be an array of length `DimensionMetaData.size + 1`, and the comment improved to document this.

Reply:
I agree with @matthew-d-jones.
Also, I think the comment for size should be something like `Number of histogram bins`.
Generally, I don't think the `for this dimension` is needed, the main comment implies this clearly.

Reply:
Yes, I agree too with @matthew-d-jones , for the size field we can rename it to something more commonly used like length.